Dockerfiles/testrail_apache/Dockerfile

```diff
@@ -98,6 +99,19 @@ RUN ( cd /tmp && \
       ./configure && \
       make install \
     )
+
+RUN ( cd /tmp && \
+      apt-get update && \
+      apt install -y libnss3 libatk1.0-0 libxcomposite1 libxdamage1 libxrandr2 libgbm1 libxkbcommon0 libasound2 libatk-bridge2.0-0 libxfixes3 && \
+      wget https://storage.googleapis.com/chrome-for-testing-public/131.0.6778.69/linux64/chrome-headless-shell-linux64.zip && \
+      unzip "chrome-headless-shell-linux64.zip" && \
```

Comment:
It needs to always fetch latest version. it can be done with API, where it fetches url for latest version

Reply:
with
wget -qO- https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_STABLE

this will send the ouput to stdout